packages/data-store-web/scripts/publish.sh

```diff
@@ -0,0 +1,3 @@
+yarn build
+npm publish --access public
+
```

Comment:
wont let me comment below but `packages/data-store-web/setup-tests.ts` is an empty file